events/misakiReady.js

```diff
@@ -3,7 +3,7 @@ const Event = require(`${process.cwd()}/base/Event.js`);
 module.exports = class extends Event {
 
   async run() {
-    await this.client.methods.util.wait(500);
+    await this.client.methods.util.wait(750);
```

Comment:
Why was this changed from `1000`? Seems like you're creeping back up to that.

Reply:
500ms was too short for command loading, 1 second is too long and 750 is just fine until i rewrite command and event loading